### DISCUSSION TITLE
tuiButton vs tuiIconButton

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#9512] [Q&A] [answered]
Hi!

Is there any disadvantage now using  `tuiButton` instead of `tuiIconButton`, even if the button only has an icon? I know that in older version there was some margin added to icon (`.t-left`) which made it inconsistent to use `tuiButton` only with an icon. But now it seems to work with new icon handling and they look completely the same...

**Comment by waterplea** [comment, marked as answer]
Yes, we were also thinking if we still need `tuiIconButton` since now it all fits right out of the box. We might reconsider it by v5, but currently an added benefit of `tuiIconButton` is that it has `font-size: 0` so you can have an icon button with text in it that won't be visible to you, but will make the button accessible for assistive technologies.

**Reply by loxy** [reply]
Good point! Thank you!
